test/e2e/kubevirt.go

```diff
@@ -1064,15 +1068,39 @@ if [ "$ipv6" != "" ]; then
 	iperf3 -s -D --bind $ipv6 --logfile /tmp/test_${ipv6}_iperf3.log
 fi
 `
-		createIperfServerPods = func(nodes []corev1.Node, netConfig networkAttachmentConfig) ([]*corev1.Pod, error) {
-			annotations := map[string]string{}
-			if netConfig.role != "primary" {
-				annotations["k8s.v1.cni.cncf.io/networks"] = fmt.Sprintf(`[{"name": %q}]`, netConfig.name)
+		nextIPs = func(idx int, subnets []string) ([]string, error) {
+			var ips []string
+			for _, subnet := range subnets {
+				ip, ipNet, err := net.ParseCIDR(subnet)
+				if err != nil {
+					return nil, err
+				}
+				for _ = range idx {
+					ip = iputils.NextIP(ip)
+				}
+				ipNet.IP = ip
+				ips = append(ips, ipNet.String())
 			}
+			return ips, nil
+		}
+		createIperfServerPods = func(nodes []corev1.Node, netConfig networkAttachmentConfig, staticSubnets []string) ([]*corev1.Pod, error) {
```

Comment:
perhaps staticSubnets-->staticNodeSubnets ? it's not clear right now that each static subnet is a different node.

Reply:
What do you mean? this is a layer2 subnet independant of the node.

Reply:
What I mean is - it looks like each string in the `staticSubnets` string list is meant for a different node:
```
			for i, node := range nodes {
...
			    staticIPs, err := nextIPs(i, staticSubnets)
....
```

so maybe it's worth to mention it in the param name, like `staticNodeSubnets`

Reply:
We are using the loop to create a pod per node, but the subnet has nothing to do with the nodes they are independent of the node, what we have per node is an staticIP assigned to the created pod.